mailet/api/src/test/java/org/apache/mailet/AttributeValueTest.java

```diff
@@ -639,18 +640,6 @@ void fromJsonStringShouldThrowOnMissingValueField() {
             .isThrownBy(() -> AttributeValue.fromJsonString("{\"serializer\":\"MapSerializer\"}"));
     }
 
-    @Test
-    void valueAsShouldCastValueWhenRightType() {
-        assertThat(AttributeValue.of("value").valueAs(String.class))
-            .contains("value");
-    }
-
-    @Test
-    void valueAsShouldReturnEmptyWhenWrongType() {
```

Comment:
why did not you copy these tests in AttributeUtilsTests?

Reply:
It's a pure revert, maybe some tests can be added at `AttributeValue`'s level, but that's another point